src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock"
+        class="card__image"
+      >
+
+      <div class="card__objects">
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h2 class="card__code">
+          Product code: 195434
+        </h2>
+
+        <div class="card__display card__display--flex-b24">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <h3 class="card__review">
+            Reviews: 5
+          </h3>
+        </div>
+
+        <div class="card__display card__display--flex-b16">
+          <h4 class="card__price">
+            Price:
+          </h4>
+
+          <h5 class="card__cost">
+            $2,199
+          </h5>
+        </div>
+
+        <a
+        href="#"
+        data-qa="hover"
+        class="card__button"
+        >
```

Comment:
don't forget about the indents between elements of different nesting levels
```suggestion
        <a
          href="#"
          data-qa="hover"
          class="card__button"
        >
```